packages/astro/src/cli/throw-and-exit.ts

```diff
@@ -20,7 +20,7 @@ export async function throwAndExit(cmd: string, err: unknown) {
 
 	const errorWithMetadata = collectErrorMetadata(createSafeError(err));
 	telemetryPromise = telemetry.record(eventError({ cmd, err: errorWithMetadata, isFatal: true }));
-	errorMessage = formatErrorMessage(errorWithMetadata);
+	errorMessage = formatErrorMessage(errorWithMetadata, true);
```

Comment:
Non-blocking, but it'd be nice to move to an options object like `{ verbose: true }` or `{ showFullStacktrace: true }` to avoid an unlabelled boolean argument.